packages/exo/src/exo-tools.js

```diff
@@ -67,16 +164,15 @@ const defendSyncArgs = (syncArgs, methodGuardPayload, label = undefined) => {
  */
 const defendSyncMethod = (method, methodGuardPayload, label) => {
   const { returnGuard } = methodGuardPayload;
+  const isRawReturn = isRawGuard(returnGuard);
+  const redactConfig = redactRawArgs(methodGuardPayload);
   const { syncMethod } = {
     // Note purposeful use of `this` and concise method syntax
     syncMethod(...syncArgs) {
-      const realArgs = defendSyncArgs(
-        harden(syncArgs),
-        methodGuardPayload,
-        label,
-      );
+      // Only harden args and return value if not dealing with a raw value guard.
+      const realArgs = defendSyncArgs(syncArgs, redactConfig, label);
       const result = apply(method, this, realArgs);
-      mustMatch(harden(result), returnGuard, `${label}: result`);
+      isRawReturn || mustMatch(harden(result), returnGuard, `${label}: result`);
```

Comment:
we usually use the `||` like this for an assertion, suggesting that left expression is supposed to be true. that's not the case here so consider,
```suggestion
      if (!isRawReturn) {
        mustMatch(harden(result), returnGuard, `${label}: result`);
      }
```